### ISSUE TITLE
Notification is not destroyed in Samsung Galaxy S21 plus lock screen

### ISSUE THREAD

**Issue by shantanupks** [ghenry22/cordova-music-controls-plugin#72]
Hi, 
we are using this plugin to show media controls notification, but we are seeing some issues in destroying the notification in Samsung OS (model: Samsung Galaxy S21 plus).

Even after calling `MusicControls.destroy(onSuccess, onError);`, we are getting success callback, But still the notification can be seen on Lock screen 

We see similar issues reported here - https://us.community.samsung.com/t5/Note20/Can-t-remove-media-controls-from-lock-screen-after-One-UI-3-0/td-p/1628685 

Exact same call works in other android devices.

We get notification like this on lock screen after `MusicControls.create({...})` call

![Success Create](https://user-images.githubusercontent.com/37309875/135474185-8611932d-1a11-4982-a091-e2c199395e6f.JPG)

However when we call  `MusicControls.destroy(onSuccess, onError);`, we see a different kind of notification (the config object in create does have these properties set)

![OS Create](https://user-images.githubusercontent.com/37309875/135474206-f3dc93c9-b546-4fe3-b9e9-3e500fecb5f8.JPG)

it looks like OS itself is creating default notification after destroy() call? (just a guess).

Any workaround for this would be appreciated, Thanks!

**Comment by srividya-sharma** [comment]
We are also facing similar issue where the Music Controls are shown even after the app is force closed. 
Sometimes, the controls go away when app is closed, but reappear after few seconds...
"it looks like OS itself is creating default notification after destroy() call? (just a guess)." -- seems to be the case for me also.
Were you able to find a solution or workaround for this?

**Comment by shantanupks** [comment]
> 
Yeah, OS itself is creating it. We are referring already open PR for this - https://github.com/ghenry22/cordova-plugin-music-controls2/pull/61 
We were not able to find any permenant solution though!
